akka-actor-testkit-typed/src/test/java/akka/actor/testkit/typed/javadsl/BehaviorTestKitTest.java

```diff
@@ -123,72 +122,102 @@ public Log(String what) {
     }
   }
 
-  public interface Action {
-  }
+  public interface Action {}
 
   private static Behavior<Action> childInitial = Behaviors.ignore();
 
   private static Props props = Props.empty().withDispatcherFromConfig("cat");
 
-  private static Behavior<Command> behavior = Behaviors.receive(Command.class)
-    .onMessage(SpawnChildren.class, (context, message) -> {
-      IntStream.range(0, message.numberOfChildren).forEach(i -> {
-        context.spawn(childInitial, "child" + i);
-      });
-      return Behaviors.same();
-    })
-    .onMessage(SpawnChildrenAnonymous.class, (context, message) -> {
-      IntStream.range(0, message.numberOfChildren).forEach(i -> {
-        context.spawnAnonymous(childInitial);
-      });
-      return Behaviors.same();
-    })
-    .onMessage(SpawnChildrenWithProps.class, (context, message) -> {
-      IntStream.range(0, message.numberOfChildren).forEach(i -> {
-        context.spawn(childInitial, "child" + i, message.props);
-      });
-      return Behaviors.same();
-    })
-    .onMessage(SpawnChildrenAnonymousWithProps.class, (context, message) -> {
-      IntStream.range(0, message.numberOfChildren).forEach(i -> {
-        context.spawnAnonymous(childInitial, message.props);
-      });
-      return Behaviors.same();
-    })
-    .onMessage(CreateMessageAdapter.class, (context, message) -> {
-      context.messageAdapter(message.clazz, message.f);
-      return Behaviors.same();
-    })
-    .onMessage(SpawnWatchAndUnWatch.class, (context, message) -> {
-      ActorRef<Action> c = context.spawn(childInitial, message.name);
-      context.watch(c);
-      context.unwatch(c);
-      return Behaviors.same();
-    })
-    .onMessage(SpawnAndWatchWith.class, (context, message) -> {
-      ActorRef<Action> c = context.spawn(childInitial, message.name);
-      context.watchWith(c, message);
-      return Behaviors.same();
-    })
-    .onMessage(SpawnSession.class, (context, message) -> {
-      ActorRef<String> session = context.spawnAnonymous(Behaviors.receiveMessage( m -> {
-        message.sessionHandler.tell(m);
-        return Behaviors.same();
-      }));
-      message.replyTo.tell(session);
-      return Behaviors.same();
-    })
-    .onMessage(KillSession.class, (context, message) -> {
-      context.stop(message.session);
-      message.replyTo.tell(Done.getInstance());
-      return Behaviors.same();
-    })
-    .onMessage(Log.class, (context, message) -> {
-      context.getLog().info(message.what);
-      return Behaviors.same();
-    })
-    .build();
-
+  private static Behavior<Command> behavior =
+      Behaviors.receive(Command.class)
+          .onMessage(
+              SpawnChildren.class,
+              (context, message) -> {
```

Comment:
It would be good if the formatter didn't enforce quite so many linefeeds everywhere...

Reply:
I don't like it either, but google's style is configuration-less...

refs : https://github.com/playframework/playframework/issues/8109

Reply:
The only way is fallback to old sbt-java-formatter if you insistent it.

Reply:
@gmethvin What's your experience about it?

Reply:
Yeah, not sure it is worth it then. I guess Java ppl are used to wide lines and many linefeeds anyway.

Reply:
But I have to say the Alibaba's Java code style is 4 spaces too ..., I think that's fine, If we don't let it as 4 spaces, we could not publish with our CI and will be recorded which have a bad effect on our bouns....hard life.

Back to this issue, it's self, if the eclipse' formatter is great, then we could fall back to it. hand roll formatting is a very bad thing in any way. And IIRC, Alpakka it's using the new version now.
https://github.com/akka/alpakka/blob/master/project/plugins.sbt#L14

For a day to day Java developer, even my self like the 2 spaces thing, but most Java developers do not buy it. @2m Is there any particular reason that Alpakka does not use the old formatter but  Google's style?

Reply:
refs:https://github.com/akka/alpakka/pull/1049/files#diff-aec4792e35f6f2f944bd68efb9916715

Reply:
We need to double check that the java samples in the docs still look ok if we go with that though.

Reply:
The formatter was changed to google's here: https://github.com/sbt/sbt-java-formatter/pull/23/files

One motivating factor was that before the change, sbt-java-formatter plugin did not have any defaults. Therefore quite extensive xml file was required in every project that used the formatter plugin.

Reply:
yes, all configuration should be done through the xml configuration, the new one works just out of box.